### PR TITLE
Add french translation

### PR DESCRIPTION
**Pull request by BoboTiG**
This is a downstream patch of https://github.com/sduduzog/slim-launcher/commit/4d08abb7da4c08b30083f6877a42fc2a905ea830.

### CODE REVIEW

**Comment by BoboTiG** [comment]
Should I fix "AAPT: error: unescaped apostrophe in string" errors?

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        <item>24 heures</item>
+        <item>12 heures</item>
+    </string-array>
+
+    <string-array name="quick_button_array">
+        <item>Afficher</item>
+        <item>Cacher</item>
+    </string-array>
+
+    <string name="choose_time_format_dialog_title">Format de l'heure</string>
```
Comment (jkuester) [inline app/src/main/res/values-fr/strings.xml:22]:
```suggestion
    <string name="choose_time_format_dialog_title">Format de l\'heure</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    <string name="main_fragment_options">Options</string>
+    <string name="options_fragment_about_slim">À propos d'Unlauncher</string>
```
Comment (jkuester) [inline app/src/main/res/values-fr/strings.xml:24]:
```suggestion
    <string name="options_fragment_about_slim">À propos d'\Unlauncher</string>
```

File: app/src/main/res/values-fr/strings.xml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    <string name="options_fragment_device_settings">Paramètres Système</string>
+    <string name="options_fragment_change_theme">Changer le Thème</string>
+    <string name="options_fragment_choose_time_format">Format de l'Heure</string>
```
Comment (jkuester) [inline app/src/main/res/values-fr/strings.xml:27]:
```suggestion
    <string name="options_fragment_choose_time_format">Format de l\'Heure</string>
```

**Review by jkuester — CHANGES_REQUESTED** [review]
Thank you for the translation!

Can you try adding these slashes I have suggested into the file?  I think the translation will technically work the way you have entered it, but some frameworks can get confused by those apostrophes (since in a lot of programming languages that represents the beginning/end of the text) and so the best-practice is to resolve the error.  The `\` character I am suggesting is known as an "escape character" which means it is just there to identify to the code that the following character (in this case the apostrophe) should be interpreted as plain text and not as indicating the beginning/end of the entry.  I have tested this locally and verified that the `\` escape character is not actually displayed to the user when running Unlauncher with the French locale.

**Review by jkuester — APPROVED** [review]

**Pull request merged**